### ISSUE TITLE
tf2onnx 1.14 - wrong conversion of BatchNormalization

### ISSUE THREAD

**Issue by svobora** [onnx/tensorflow-onnx#2150]
using "tf2onnx.convert.from_function" on lambda with Keras model,

tf2onnx 1.13 - works ok, batch normalization layer has members "mean" and "var"
tf2onnx 1.14 - batch normalization layer has members "input_mean" and "input_var" instead of "mean" and "var".

.onnx file converted with 1.14 thus cannot be used to generate TensorRT file.

![image](https://user-images.githubusercontent.com/17670772/228452109-f87925f2-280c-4c50-ab85-1cb756d5c7fd.png)

**Comment by fatcat-z** [comment]
This is by design.

For BatchNormalization op, the names of these 2 inputs were changed since opset 14. You can check the difference between opset 9 and 14 from [here](https://onnx.ai/onnx/operators/text_diff_BatchNormalization_9_14.html).

Since tf2onnx 1.14, we changed the default opset from 13 to 15 so such changes happend to this op. If you want to keep the previous behavior, please use ['--opset' parameter](https://github.com/onnx/tensorflow-onnx#--opset) while converting a model to ONNX.
